src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,37 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import cn from 'classnames';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  selectedTodoId: number;
+  setSelectedTodoId: React.Dispatch<React.SetStateAction<number>>
+  todos: Todo[];
+};
+
+export const TodoModal: React.FC<Props> = ({
+  selectedTodoId,
+  setSelectedTodoId,
+  todos,
+}) => {
+  const [user, setUser] = useState<User>();
+
+  const selectedTodo = todos.find(todo => todo.id === selectedTodoId);
```

Comment:
will be better to store null here if you don't find todo add show corresponding message about it